### DISCUSSION TITLE
Version@3.5.3

### DISCUSSION THREAD

**Discussion by khengyun** [khengyun/autonext#26] [Announcements]
Download For Window: [click here](https://github.com/khengyun/autonext/archive/refs/tags/v3.5.3.zip)
Download For Linux: [click here](https://github.com/khengyun/autonext/archive/refs/tags/v3.5.3.tar.gz)

Feature:

Presentation Grading
Group Members Grading
Support Chromium and Microsoft Edge

[NEW]: 
Support New Edunext

[FIX]:
Port error

v3.5.3

https://github.com/khengyun/autonext/assets/78076796/80492c3f-2801-4ef4-a75c-e47fa8e9981e

<hr /><em>This discussion was created from the release <a href='https://github.com/khengyun/autonext/releases/tag/v3.5.3'>Version@3.5.3</a>.</em>

**Comment by khengyun** [comment]
This version supports updating api from outside 
https://github.com/khengyun/autonext/blob/a05f63f64debfb950456a469fe8a36dfcf24d886/background_script/apicallpack.json

**Comment by TwoP62** [comment]
![image](https://github.com/khengyun/autonext/assets/134353561/e017eacd-f77a-473e-a020-88a6dc3f5e0b)

có thể giúp mình với không ạ. mình cài đặc giống như clip nhưng nó ko hoạt động ancom45678@gmail.com
